packages/devtools_app/lib/src/flutter/table.dart

```diff
@@ -52,13 +56,10 @@ class _TreeTableState<T extends TreeNode<T>> extends State<TreeTable<T>>
   /// The width of each column in this table, including the
   /// appropriate indentation for nested rows.
   ///
-  /// This is computed in [_computeTableWidth].
+  /// This is computed in [_computeColumnWidths].
   List<double> columnWidths = [];
-  double get tableWidth => columnWidths.reduce((x, y) => x + y);
-
-  AnimationController _resizeController;
-  Animation<double> _resizeAnimation;
-  Tween<double> _widthTween;
+  double get tableWidth =>
+      columnWidths.isEmpty ? 0.0 : columnWidths.reduce((x, y) => x + y);
```

Comment:
ultra nit: not sure this code reads better than the simpler straight line code
```dart
double get tableWidth {
  double width = 0;
  for (var width in columnWidths)
    width += columnWidth;
  return width;
}
```

Reply:
Good suggestion.  Done.